docs/Introduction.md

```diff
@@ -0,0 +1,74 @@
+
+Manta's underlying logic is to mint private coins/tokens with stable coins and other base tokens on the contract at a minting value ratio of 1:1. Apart from that, a decentralized private swap platform can support the exchange of these privacy tokens.
+
+![Manta Overview](/img/manta_overview.png)
+
+The Manta Network mainly consists of two protocols: **the decentralized private payment protocol** and the **decentralized private token exchange protocol.**
+
+## Products
+
+### MantaPay: Privacy Preserving Multi-Asset Decentralized Payment
+
+Is based on zk-SNARK. It allows users to swap Polkadot and its Parachain tokens with corresponding private tokens, pay with privacy tokens, and redeem base coins from private tokens.
+
+### MantaSwap: Privacy Preserving Decentralized Exchange
+
+DAX protocol is based on AMM and zk-SNARK. Users can anonymously trade private versions of tokens on the platform between each other. The price formation method of the decentralized private token exchange protocol is consistent with the mainstream AMM. The revenue model of Manta tokens is based on the rebate and redeem of swap usage fees and transaction fees. In the future, the team will also develop products like synthetic asset stable coins based on Manta.
+
+We also plan to integrate Manta products into decentralized private lending protocols. Decentralized private lending is a decentralized non-custodial lending platform where users can deposit anonymously and borrow private coins on the platform. Decentralized private synthetic asset protocols and other DeFi services are included in the future roadmap.
+
+## Problems solved and application scenarios in Web3 by Manta
+
+Although the current distributed ledger has brought a new payment model without the participation of third-party financial institutions, most records of the current cryptocurrency transactions are publicly available due to the transparency of blockchain technology. Public transactions can be viewed by competitors or third-party organizations, which affects business interests. At an individual level, public transactions can be taken advantage of by advertisers and criminals.
+
+Although privacy-preserving cryptocurrencies such as ZCash exist, the status quo has two significant problems. High price volatility is the first problem, which affects practical use. The lack of scalability is the second problem, which limits use to only simple payments. The exchange of privacy-preserving coins and other cryptocurrencies, including other privacy-preserving coins, is still unsolved — that still relies on centralized exchanges and OTC.
+
+Manta Network addresses the above two pain points. First, **the decentralized privacy-preserving payment protocol** based on **zk-SNARK** supports Polkadot and Substrate Parachain assets to their corresponding private pairs. In this way, users can use popular assets like stable coins, renBTC, etc., while simultaneously benefiting from the anonymity of privacy coins.
```

Comment:
```suggestion
Manta Network solves both of these pain points. First, **MantaPay** supports Polkadot and Substrate Parachain assets with corresponding private tokens. This allows users to transact in popular assets like stable coins, renBTC, etc., while simultaneously benefiting from the anonymity of privacy coins. Second, **MantaSwap** can realize decentralized transactions between private coins, which is similar to Uniswap and Curve, and ensure transaction addresses' **privacy**.
```

By this point in the doc, Manta Pay and MantaSwap have already been introduced, so the point of this paragraph is just to highlight how Manta does better than any other existing platform.